fr/core/Execution flow.md

```diff
@@ -0,0 +1,82 @@
+@(Beluga)[Documentation, Technical]
+
+## Flux d'execution de Beluga
+
+### Accès à Beluga
+
+Beluga ne peut être directement instancié, aussi il doit être récupéré au travers de la methode ``getInstance``. Ceci permet de n'effectuer qu'une seule fois l'initialisation du framework, lors du premier appel de la methode. Aussi, cette methode prend en paramètre une connection vers une base de données, laissant ainsi le choix au développeur de laisser, ou non, Beluga se connection à la base de donnée.
```

Comment:
* méthode pas methode (cette erreur apparaît plusieurs fois d'ailleurs).
 * connexion pas connection.

"Beluga se connection à la base de donnée": not ok